src/utils/ruleCalls.js

```diff
@@ -1,68 +1,34 @@
 import axios from "axios";
 
-export const listRules = async () => {
-  const response = await axios.get(`${process.env.API_URL}/bounce_rules`, {});
-  if (response.status === 200) {
-    return response;
-  }
-  throw new Error("Error retrieving all rules");
-};
+export const listRules = async () =>
+  axios.get(`${process.env.API_URL}/bounce_rules`).catch(err => err.response);
```

Comment:
can we put the `.catch` on the line below for consistency?

Reply:
Prettier is formatting it here on save :c